whois-client/src/main/java/net/ripe/db/whois/api/rest/mapper/ValidListXmlAdapter.java

```diff
@@ -0,0 +1,20 @@
+package net.ripe.db.whois.api.rest.mapper;
+
+import jakarta.xml.bind.annotation.adapters.XmlAdapter;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ValidListXmlAdapter extends XmlAdapter<String, List<String>> {
+
+    @Override
+    public List<String> unmarshal(String value) {
+        String[] parts = value.split("\\n{2,}");
```

Comment:
What happens when warning messages itself has a /n in it but still a single warning?

Reply:
two /n are required to match the regex.
Anyway with the change requested by Ed this class is not longer necessary